### ISSUE TITLE
Skylake: panic on disconnecting external display

### ISSUE THREAD

**Issue by AMDmi3** [FreeBSDDesktop/freebsd-base-graphics#153]
Dell Latitude E7470, drm-next branch, latest ports.
```
% sysctl hw.model
hw.model: Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz
% uname -a
FreeBSD marakasov 12.0-CURRENT FreeBSD 12.0-CURRENT #0 d71bcb11d(drm-next): Wed Jun 21 17:14:15 MSK 2017     root@marakasov:/usr/obj/usr/src/sys/GENERIC  amd64
```
Disconnecting external display the second time leads to panic. (E.g. I can disconnect it, connect it again, and when I disconnect it again it panics). Traceback pending.

**Comment by hselasky** [comment]
Can you try the very latest drm-next ? Backtrace would be appreciated.

**Comment by AMDmi3** [comment]
This is almost latest drm-next (previous to latest commit), or do you mean that it may be fixed efeb728?

**Comment by AMDmi3** [comment]
As with #152, this was fixed by removing xorg.conf
